machine-learning/lambda-v2/README.md

```diff
@@ -2,14 +2,20 @@
 
 These example components are used to run sample image classification and object detection inferences with AWS lambda.
 
+
 Image Classification 
- - com.lambda.DLRImageClassification 
+ - com.lambda.DLRImageClassification(No container  mode)
```

Comment:
1. Add space between component name and parenthesis.
2. Remove extra space in `No container  mode`